frontend/src/components/ui/WorkspaceTabs.jsx

```diff
@@ -2,11 +2,16 @@ import { useEffect, useState } from "react";
 import { Tabs, TabList, Tab, TabPanels, TabPanel } from "@carbon/react";
 import { workspaceAtom } from "@/atoms/pipelineAtom";
 import { useImmerAtom } from "jotai-immer";
+import { useAtom } from "jotai";
+import { drawflowEditorAtom } from "@/atoms/drawflowAtom";
 
 export default function WorkspaceTabs() {
   const [workspace, setWorkspace] = useImmerAtom(workspaceAtom);
   const [renderedTabs, setRenderedTabs] = useState([]);
   const [selectedIndex, setSelectedIndex] = useState(0);
+  const [zoom_levels, setZoomLevels] = useState({});
```

Comment:
I would call this `zoomLevels` to be consistent with our naming convention.

Reply:
changed it to zoomLevels